### PR TITLE
add code

### PR DESCRIPTION
**Pull request by murixiyang**
I add a code to the prime document

### CODE REVIEW

**Review by Lucanchvans — COMMENTED** [review]
This is comment 1

**Review by Lucanchvans — CHANGES_REQUESTED** [review]
You need change the function name to **print_prime**

File: prime.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+def is_prime(num):
+    if num <= 1:
+        return False
+    for i in range(2, int(num**0.5) + 1):
+        if num % i == 0:
+            return False
+    return True
+
+def print_primes(n):
```
Comment (Lucanchvans) [inline prime.py]: Change the name
```suggestion
def print_prime(n):
```

Comment (murixiyang) [reply prime.py]: Got it

**Pull request merged**